scripts/README.md

```diff
@@ -98,7 +98,7 @@ please do not add new entries to these files. PRs removing (the need for) entrie
 - `count-trans-deps.py`, `import-graph-report.py` and `import_trans_difference.sh` produce various
   summaries of changes in transitive imports that the `PR_summary` message incorporates.
 - `zulip_emoji_merge_delegate.py` is called every hour by a Github action cronjob.
-  It looks through the latest 200 zulip posts: if a message mentions a PR that is delegated, or sent to bors, or merged,
```

Comment:
optionally, you could make this more precise: it looks through the latest 200 posts on *each relevant* channel.